### PR TITLE
Update sigs.yml

### PR DESCRIPTION
**Pull request by musingvirtual**
Added details for the new ContribEx SIG

### CODE REVIEW

File: sigs.yml [hunk 1]
```diff
@@ -659,4 +659,24 @@
...
\ No newline at end of file
+        github: jpkrohling
+    - name: Contributor Experience
+    meeting: TBA
+    notes:
+      type: gDoc
+      value: 1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0
+    chat:
+      - type: slack
+        name: '#otel-sig-contribex'
```
Comment (mx-psi) [inline sigs.yml:670]: Should this be #otel-contributor-experience instead?

Comment (musingvirtual) [reply sigs.yml]: @mx-psi fixed, thanks

**Review by mx-psi — APPROVED** [review]

File: sigs.yml [hunk 2]
```diff
@@ -659,4 +659,24 @@
...
+    sponsors:
+      - name: 'Juraci Paixão Kröhling'
+        github: jpkrohling
+      - name: 'Severin Neumann'
+        github: svrnm
+    gcLiaison:
+      - name: 'Juraci Paixão Kröhling'
+        github: jpkrohling
+      - name: 'Severin Neumann'
+        github: svrnm
```
Comment (svrnm) [inline sigs.yml:682]: only one of us should be the liaison, cc @jpkrohling

File: sigs.yml [hunk 3]
```diff
@@ -659,4 +659,24 @@
...
+        github: jpkrohling
+    - name: Contributor Experience
+    meeting: TBA
+    notes:
+      type: gDoc
+      value: 1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0
+    chat:
+      - type: slack
+        name: '#otel-contributor-experience'
+        id: C07C5BC6Q6L
```
Comment (svrnm) [inline sigs.yml:671]:
```suggestion
        id: C06TMJ2R0SK
```

**Comment by svrnm** [comment]
make sure that you run `make table-generation` locally and upload the updated README.md

**Review by arminru — APPROVED** [review]

File: sigs.yml (hunk 2 above)
Comment (jpkrohling) [reply sigs.yml:682]: I'm fine being the liaison

**Review by jpkrohling — APPROVED** [review]
Approving preemptively, even though a change is needed to the liaison.

File: sigs.yml (hunk 2 above)
Comment (musingvirtual) [reply sigs.yml:682]: Thanks both! We are going to have to update this doc again once we agree on a meeting time, so I might just leave this PR open for a day or two and wait to re-run all the checks once we have that as well so I'm not doing it twice.

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**